apidoc/Titanium/UI/Slider.yml

```diff
@@ -394,6 +402,7 @@ properties:
   - name: tintColor
     summary: The color shown for the portion of the progress bar that is filled.
     type: [String, Titanium.UI.Color]
+    platforms: [iphone, ipad, android, macos]
```

Comment:
This should usually not be necessary, since the default is set to match all platforms

Reply:
the Android icon wasn't visible here before, now it is:

![Screenshot_20220731_164158](https://user-images.githubusercontent.com/4334997/182031617-3c384de0-1cb2-482b-830e-9d0148f3443c.png)

trackTintColor was visible already because it had the platforms array.

Reply:
perhaps it is a different setting but this was the PR before:

https://github.com/tidev/titanium_mobile/pull/11074/files#diff-53e21b71d097276c9bb920965872bc92c416bcc0bbae08c894d8b25029313437R394-R405

and Android didn't show up for `tintColor`